lisa/sut_orchestrator/hyperv/get_assignable_devices.ps1

```diff
@@ -0,0 +1,216 @@
+param (
```

Comment:
Can you convert it to Python? The mixed scripts are hard to maintain and troubleshooting. Currently, LISA has only 2 PowerShell scripts, one is to enable SSH before connection built, another is for reliability to generate serial log from named pipes.

Reply:
Sure, I will convert this in python.

I have integrated the already existing script in Microsoft document for DDA device. Idea was to not change much in the file.

Reply:
@squirrelsc This seems a big task. Can we take it as incremental changes?

Reply:
Please finish it in this PR. It brings complex and hard to maintain parts in LISA. If you want to do integration test early, you can create a dev branch in microsoft/lisa to unblock work.

Reply:
@squirrelsc I removed powershell file and added python script now. Please check latest push.